modules/admin-top-bar/assets/scss/_index.scss

```diff
@@ -0,0 +1,119 @@
+$top-bar-height: 50px;
+
+#e-admin-top-bar {
+	background: #fff;
+	box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.0394871);
+	position: absolute;
+	top: 0;
+	right: 0;
+	width: calc(100% - 160px);
+	z-index: 9999;
```

Comment:
Don't use value like `50px` `9999`, `#fff`. use pre-defined units and variables.

Reply:
I have changed the colors to colors that exists in our variables file and used the z-index variable but I am not sure about the 50px, this values are not pre defined anywhere at least I am not aware of.

Reply:
It's a unit from the Design System, not a variable. @joshmarom can you advise?

Reply:
Josh said I cant use it as a framework, it was something very specific to a feature he worked on.